lms/templates/ux/reference/bootstrap/course-skeleton.html

```diff
@@ -0,0 +1,79 @@
+## Override the default styles_version to use Bootstrap
+<%! main_css = "css/bootstrap/lms-main-bootstrap.css" %>
+
+<%page expression_filter="h"/>
+<%inherit file="/main.html" />
+
+<%block name="pagetitle">Course Skeleton</%block>
+
+<%block name="bodyclass">course-skeleton</%block>
+
+<!-- Need jquery download to assure that dropdown works TODO: pull bootstrap in through webpack -->
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
```

Comment:
@andy-armstrong - here is the hacky jquery solution, I think you might be right about tossing it in the common.py base_vendor compile.